File: JS/src/models/item.js
class Item {
    constructor(itemId, itemName, itemHeight, itemWidth, itemLength, startDate, endDate, isActive = true) {
        this.itemId = itemId;
        this.itemName = itemName;
        this.itemHeight = itemHeight;
        this.itemWidth = itemWidth;
        this.itemLength = itemLength;
        this.startDate = new Date(startDate);
        this.endDate = new Date(endDate);
        this.isActive = isActive;
    }
}

File: WarehouseProjectDotNET/WarehouseProject/Models/Item.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WarehouseProject.Models
{
    public class Item
    {
        public int ItemId { get; set; }
        public string ItemName { get; set; }
        public float ItemHeight { get; set; }
        public float ItemWidth { get; set; }
        public float ItemLength { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public bool IsActive { get; set; }
    }
}
